scanner/cmd/scannerctl/main.go

```diff
@@ -29,7 +31,15 @@ func main() {
 		utils.CrashOnError(os.Setenv(mtls.CertFilePathEnvName, filepath.Join(*certsPath, mtls.ServiceCertFileName)))
 		utils.CrashOnError(os.Setenv(mtls.KeyFileEnvName, filepath.Join(*certsPath, mtls.ServiceKeyFileName)))
 	}
-
+	// Extract basic auth username and password.
+	var username, password string
+	if *basicAuth != "" {
+		auth := strings.SplitN(*basicAuth, ":", 2)
```

Comment:
nit: consider using `strings.Cut`, as it is meant for the special case of `strings.SplitN(2)`